src/os/ui/window/geoserverhelp.js

```diff
@@ -0,0 +1,79 @@
+goog.module('os.ui.window.GeoServerHelpUI');
```

Comment:
This module should be in `src/plugin/ogc/ui`, and the view in `views/plugin/ogc/ui`.